File: breun-java/Raffler.java
import java.io.IOException;
import java.util.List;
import java.util.Random;
import java.nio.file.Files;
import java.nio.file.Paths;

public class Raffler {
    public static final void main(String[] args) throws IOException {
        List<String> attendees = Files.readAllLines(Paths.get(args[0]));
        System.out.println(attendees.get((new Random()).nextInt(attendees.size())));
    }
}

File: kotlin-annabaas/src/Raffler.kt
import java.io.File
import java.io.IOException
import java.util.Random

object Raffler {
    @Throws(IOException::class)
    @JvmStatic fun main(args: Array<String>) {
        val inputFile = File(args[0])
        val attendeeList = inputFile.readLines()
        val winnerInt = Random().nextInt(attendeeList.size)
        println (attendeeList[winnerInt])
    }
}
